cognite/client/_api/data_modeling/instances.py

```diff
@@ -92,6 +94,8 @@
 
 logger = logging.getLogger(__name__)
 
+Source: TypeAlias = Union[SourceDef, View, ViewId, Tuple[str, str], Tuple[str, str, str]]
```

Comment:
Won't this be sufficient?
```suggestion
Source: TypeAlias = Union[SourceDef, View, ViewIdentifier]
```